assets/images/icons/airport.svg

```diff
@@ -1,95 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
no worry, the svgs are all still there.  The diff is going to be hard to read for this MR, due to all the changes.  I did add an `imagemin` task that will optimize the pngs and svgs, but I'm not sure that is actually necessary.